sched/semaphore/spinlock.c

```diff
@@ -36,6 +36,11 @@
 #  include <stdatomic.h>
 #endif
 
+#ifdef CONFIG_RW_SPINLOCK
+#  include <stdatomic.h>
+const static rwlock_t unlocked = RW_SP_UNLOCKED;
```

Comment:
but, it's still better to define a local variable instead global.

Reply:
Got it.